DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/AddSupervisor.cshtml

```diff
@@ -79,30 +79,27 @@
                         <input type="hidden" aria-hidden="true" id="select-sort-direction" value="Ascending" />
 
                         <form id="frmSubmit" method="post" >
-                            <div id="searchable-elements" class="nhsuk-radios">
-                                @foreach (var supervisor in Model.Supervisors)
-                                {
-                                   <div Class="searchable-element nhsuk-u-padding-top-3">
-                                    <div class="nhsuk-radios__item">
-                                        <input class="nhsuk-radios__input" id="supervisor-@supervisor.AdminID" name="SupervisorAdminID" type="radio" checked="@(supervisor.AdminID == Model.SupervisorAdminID)" value="@supervisor.AdminID" aria-describedby="supervisor-@supervisor.AdminID-item-hint">
-                                        <label class="nhsuk-label nhsuk-radios__label" for="supervisor-@supervisor.AdminID">
-                                            @supervisor.Forename @supervisor.Surname (@supervisor.CentreName)
-                                        </label>
-                                        <div class="nhsuk-hint nhsuk-radios__hint" id="supervisor-@supervisor.AdminID-item-hint">
-                                            @supervisor.Email
-                                        </div>
-                                    </div>
-                                  </div>
-                                }
-                            </div> 
-                            @if (Model.Supervisors.Count() > 0 || Model.MatchingSearchResults>0)
+                          <div class="searchable-element nhsuk-u-padding-top-3">
+                            @foreach (var supervisor in Model.Supervisors)
                             {
-                                <button id="btnAddSupervisor" class="nhsuk-button nhsuk-u-margin-top-3" type="submit">
-                                    Next
-                                </button>
+                              <div class="nhsuk-radios__item">
+                                <input class="nhsuk-radios__input" id="supervisor-@supervisor.AdminID" name="SupervisorAdminID" type="radio" checked="@(supervisor.AdminID == Model.SupervisorAdminID)" value="@supervisor.AdminID" aria-describedby="supervisor-@supervisor.AdminID-item-hint">
+                                <label class="nhsuk-label nhsuk-radios__label" for="supervisor-@supervisor.AdminID" style="display:block; word-break:break-all; word-wrap:break-word">

```

Comment:
Shouldn't we move this into css to avoid inline styling / copying pasting?

Reply:
Done.